templates/create-daml-app/ui/src/config.ts

```diff
@@ -14,10 +14,13 @@ export const deploymentMode: DeploymentMode =
   ? DeploymentMode.PROD_DABL
   : DeploymentMode.PROD_OTHER;
 
-export const ledgerId =
+// Decide the ledger ID based on the deployment mode first,
+// then an environment variable, falling back on the sandbox ledger ID.
+export const ledgerId: string =
   deploymentMode === DeploymentMode.PROD_DABL
   ? window.location.hostname.split('.')[0]
-  : 'create-daml-app-sandbox';
+  : process.env.REACT_APP_LEDGER_ID
+  ?? 'create-daml-app-sandbox';
```

Comment:
I don't know how the fixities of the ternary operator and `??` compare. Thus, I'd prefer to make it more explicit:
```suggestion
  : (process.env.REACT_APP_LEDGER_ID ?? 'create-daml-app-sandbox');
```

Reply:
`process.env.REACT_APP_LEDGER_ID` should never be missing, and will default to `'create-daml-app-sandbox'` as specified in `ui/.env`

Reply:
Why would you add the `??` clause in the first place then? If we add it, and I think we should, let's please make explicit where we add the default.